widgets/search-box/__tests__/search-box-test.js

```diff
@@ -182,7 +182,7 @@ describe('search-box()', () => {
         widget.getInput = sinon.stub().returns(input);
       });
 
-      it('performs the search on keyup', () => {
+      it('do not perform the search on keyup', () => {
```

Comment:
I think the name of the test is not reflecting what it does. It definitely must "perform the search on keyup", right?